### PR TITLE
[dmx] fix documentation typo

### PR DESCRIPTION
**Pull request by funkstille**
Updated Type-ID colortemperature to its correct value color_temperature

### CODE REVIEW

**Comment by funkstille** [comment]
error in Type-ID

File: bundles/org.openhab.binding.dmx/README.md [hunk 1]
```diff
@@ -194,7 +194,7 @@ The next `ON` command uses these values instead of the default (or configuration
 |-----------------|---------------------|----------------------|----------------------------------------------------|
 |brightness       |dimmer, tunablewhite |Switch, Dimmer        | controls the brightness                            |
 |color            |color                |Switch, Dimmer, Color | allows to set the color and brightness             |
-|colortemperature |tunablewhite         |Number                | allows to set the color temperature                |
+|color_temperature |tunablewhite         |Number                | allows to set the color temperature                |
```
Comment (J-N-K) [inline bundles/org.openhab.binding.dmx/README.md]:
```suggestion
|color_temperature|tunablewhite         |Number                | allows to set the color temperature                |
```

Comment (kaikreuzer) [reply bundles/org.openhab.binding.dmx/README.md]: @funkstille Could you accept @J-N-K's change suggestion? We could then merge. Thanks!

**Comment by kaikreuzer** [comment]
Thanks!

**Pull request merged**